pkg/core/store.go

```diff
@@ -204,12 +203,12 @@ func (s *StoreInfo) AllowLeaderTransferOut() bool {
 
 // EvictedAsSlowStore returns if the store should be evicted as a slow store.
 func (s *StoreInfo) EvictedAsSlowStore() bool {
-	return s.slowStoreEvicted
+	return s.slowStoreEvicted.Load() > 0
```

Comment:
Do we need to revert the order changes by last PR?

Reply:
No need